halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Plugin.java

```diff
@@ -37,18 +39,24 @@
   public Boolean enabled;
   public String manifestLocation;
 
+  @Getter(AccessLevel.NONE)
+  public Manifest manifest;
```

Comment:
Why is this `public` (and why are we generating an accessor with `AccessLevel.NONE`)?